src/components/Events/EventInfo.js

```diff
@@ -3,19 +3,23 @@ import styled from "styled-components";
 
 import  EventCard  from './EventCard'
 import Card from '../UI/Card';
+import LoadingDots from '../LoadingDots';
 
 const StyledCard = styled(Card)`
     padding: 1rem;
     margin: 2rem auto;
+    margin-bottom: 4rem;
     width: max-content;
 `;
 
 const apiUrl = process.env.REACT_APP_SERVER_URL
 
 const DisplayEventInfo = () => {
+    const [ callLoading , setCallLoading ] = useState(false);
```

Comment:
When it's a boolean state, naming `isCallLoading` would be helpful to get the idea quickly.